### PR TITLE
PS-4653 Extended mysqlbinlog doc page should only discuss --compress

### PR DESCRIPTION
**Pull request by patrickbirch**
Removed ssl information from extended_mysqlbinlog
Rewrote the compression paragraphs.
Added example

### CODE REVIEW

File: doc/source/flexibility/extended_mysqlbinlog.rst [hunk 1]
```diff
@@ -4,30 +4,14 @@
...
-   implies --ssl).
- * :option:`--ssl-capath=name` - CA directory (check OpenSSL docs, implies
-   --ssl).
- * :option:`--ssl-cert=name` - X509 cert in PEM format (implies --ssl).
- * :option:`--ssl-cipher=name` - SSL cipher to use (implies --ssl).
- * :option:`--ssl-key=name` - X509 key in PEM format (implies --ssl).
- * :option:`--ssl-verify-server-cert` - Verify server's "Common Name" in its
-   cert against hostname used when connecting. This option is disabled by
-   default.
+|Percona Server| has implemented compression support for :command:`mysqlbinlog`. This option is supported in both of the ``mysql`` and ``mysqldump`` programs. The compression option helps reduce the bandwidth use and improves the transfer speed.
```
Comment (dutow) [inline doc/source/flexibility/extended_mysqlbinlog.rst]: :command:`mysql` and  :command:`mysqldump`?

File: doc/source/flexibility/extended_mysqlbinlog.rst [hunk 2]
```diff
@@ -4,30 +4,14 @@
...
+
+In the `mysqlbinlog utility <https://dev.mysql.com/doc/refman/5.7/en/mysqlbinlog.html>`__   use either the ``-C`` or ``--compress`` option.
```
Comment (dutow) [inline doc/source/flexibility/extended_mysqlbinlog.rst]: The option is the same for the other programs too, and this implies that it's specific to this command.

Comment (patrickbirch) [reply doc/source/flexibility/extended_mysqlbinlog.rst]: Rewrote the paragraphs and removed references to other programs.

File: doc/source/flexibility/extended_mysqlbinlog.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/flexibility/extended_mysqlbinlog.rst]: fixed - removed references to other programs. Rewrote doc.

**Review by dutow — APPROVED** [review]

**Pull request merged**